### PR TITLE
updated changelog

### PR DESCRIPTION
**Pull request by SamMarkowitz**
Waiting for final version number and whether XML and new Amazon content will be in.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
 #CHANGE LOG
 
+##Version 0.?
```
Comment (Bonczidai) [inline CHANGELOG.md]: 0.9.50

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+
++ DSL Improvements
+  + Improved Documentation Style - The infile documentation of flows and operations has been changed to be more structured.
+	+ System Properties Files - System properties files now end with the .sl extension and their syntax has changed.
```
Comment (Bonczidai) [inline CHANGELOG.md]: .prop.sl

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,23 @@
...
++ CLI Enhancements
+  + Inspect - The new `inspect` command displays a file's documentation including description, inputs, outputs and results.
```
Comment (Bonczidai) [inline CHANGELOG.md]: also `load` cmd

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,23 @@
...
+	+ Default Folder - The location of the default folders for automatically importing system properties files and input files has moved to `cslang/configuration/inputs` and `cslang/configuration/properties`.
```
Comment (Bonczidai) [inline CHANGELOG.md]: we can also mention config/cslang.properties with log4j and encoding props (by default log4j can be customized from configuration.logging/log4j.properties)

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,25 @@
 #CHANGE LOG
 
+##Version 0.9.50
+
++ DSL Improvements
+  + Improved Documentation Style - The infile documentation of flows and operations has been changed to be more structured.
+	+ System Properties Files - System properties files now end with the .prop.sl extension and their syntax has changed.
```
Comment (Bonczidai) [inline CHANGELOG.md]: SP changes should on the same level with `Improved Documentation Style` right? And maybe mention that not only the SP file syntax changes bu the access style

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,25 @@
...
++ CLI Enhancements
+  + Inspect - The new `inspect` command displays a file's documentation including description, inputs, outputs and results.
+	+ List - The new `list` command lists the system properties contained in a system properties file.
+	+ Default Folder - The location of the default folders for automatically importing system properties files and input files has moved to `cslang/configuration/inputs` and `cslang/configuration/properties`.
+	+ Verbosity Level - The amount of information printed to the screen by the CLI can be changed using the `--v` flag.
+	+ Configuration - A new configuration file can be found at `cslang/configuration/cslang.properties`. The file currently includes configuration for character encoding and the location of the new logging configuration file, which is found at `cslang/configuration/logging/log4j.properties` by default.
++ Content
+  + Restructuring - The folders in the content repository have been restructured.
+	+ Additions - Content has been added in the following areas:
```
Comment (Bonczidai) [inline CHANGELOG.md]: please check the indentation in this section

**Comment by Bonczidai** [comment]
We can also mention:
- compiler error handling changes
- validate flow name and file name are identical

WDYT?

File: CHANGELOG.md (hunk 5 above)
Comment (SamMarkowitz) [inline CHANGELOG.md]: Strange. Indentation is fine in my local copy. I'll figure it out.

File: CHANGELOG.md (hunk 6 above)
Comment (SamMarkowitz) [inline CHANGELOG.md]: Here too. Fine in local copy.

**Comment by SamMarkowitz** [comment]
@Bonczidai some questions:

+ Is validating the flow name and file name part of the CLI or does that belong with the DSL changes?
+ Compiler error handling changes: I couldn't quite pinpoint what you're referring to from the GitHub chatter. Can you point me in the right direction?

**Comment by Bonczidai** [comment]
:+1:

**Pull request merged**